sql/parse/parse.go

```diff
@@ -683,6 +687,35 @@ func convertAlterIndex(ctx *sql.Context, ddl *sqlparser.DDL) (sql.Node, error) {
 	}
 }
 
+func convertAlterAutoIncrement(ddl *sqlparser.DDL) (sql.Node, error) {
+	val, ok := ddl.AutoIncSpec.Value.(*sqlparser.SQLVal)
+	if !ok {
+		return nil, ErrInvalidSQLValType.New(ddl.AutoIncSpec.Value)
+	}
+
+	var autoVal int64
+	if val.Type == sqlparser.IntVal {
+		i, err := strconv.ParseInt(string(val.Val), 10, 64)
+		if err != nil {
+			return nil, err
+		}
+		autoVal = i
+	} else if val.Type == sqlparser.FloatVal {
+		f, err := strconv.ParseFloat(string(val.Val), 10)
+		if err != nil {
+			return nil, err
+		}
+		autoVal = int64(f)
+	} else {
+		return nil, ErrInvalidSQLValType.New(ddl.AutoIncSpec.Value)
+	}
+
+	return plan.NewAlterAutoIncrement(
+		plan.NewUnresolvedTable(ddl.Table.Name.String(), ddl.Table.Qualifier.String()),
```

Comment:
Pretty sure there is a convenience function for this transformation you should use

Reply:
tableNameToUnresolvedTable

Reply:
sg